conda_smithy/templates/README.md.tmpl

```diff
@@ -71,10 +71,22 @@ Current build status
 ====================
 {%- set appveyor_badge_name = github.repo_name.replace('_', '-') %}
 {%- set appveyor_url_name = appveyor_badge_name.replace('.', '-') %}
-
+{# #}
+{%- if "circle" in ci_enabled %}
 Linux: [![Circle CI](https://circleci.com/gh/{{github.user_or_org}}/{{github.repo_name}}.svg?style=svg)](https://circleci.com/gh/{{github.user_or_org}}/{{github.repo_name}})
+{%- else %}
+Linux: ![](https://cdn.rawgit.com/{{github.user_or_org}}/{{github.repo_name}}/master/ci_support/disabled.svg)
```

Comment:
Ah sorry I misread this earlier. Yeah, we could do that. One thought though. How do we handle the fact that it may have moved location in the future? Should we point to a particular commit? Other thoughts?